src/scrapers/hapoalim.js

```diff
@@ -127,7 +127,10 @@ function getPossibleLoginResults() {
   const urls = {};
   urls[LOGIN_RESULT.SUCCESS] = [`${BASE_URL}/portalserver/HomePage`, `${BASE_URL}/ng-portals-bt/rb/he/homepage`, `${BASE_URL}/ng-portals/rb/he/homepage`];
   urls[LOGIN_RESULT.INVALID_PASSWORD] = [`${BASE_URL}/AUTHENTICATE/LOGON?flow=AUTHENTICATE&state=LOGON&errorcode=1.6&callme=false`];
-  urls[LOGIN_RESULT.CHANGE_PASSWORD] = [`${BASE_URL}/MCP/START?flow=MCP&state=START&expiredDate=null`];
+  urls[LOGIN_RESULT.CHANGE_PASSWORD] = [
+    `${BASE_URL}/MCP/START?flow=MCP&state=START&expiredDate=null`,
+    /[.]co[.]il\/ABOUTTOEXPIRE\/START/i,
```

Comment:
@esakal if you choose to include the `co.il` part in the regex expression, maybe you should include the `BASE_URL`? I'm also ok with dropping the domain part altogether.

Reply:
@eshaham I knew it will come up :) jslint has been giving me a hard time. In order to prefix the expression with `BASE_URL` I needed to use `RegExp` and thus escape the argument. 

By doing so, jslint error about `no-useless-escape`.
![image](https://user-images.githubusercontent.com/4751797/42132338-d501810e-7d1e-11e8-951e-8637be8b7c04.png)

So I tried to concat them and got `prefer-template` error from jslint instead
![image](https://user-images.githubusercontent.com/4751797/42132342-e86d907a-7d1e-11e8-89cc-4b9b41265b12.png)

Since you don't like bypassing jslint rules I tried to find a way that will satisfy jslint and this is what I come up with :) Are you familiar with an other approach I can try here which will be acceptable by jslint?